packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -63,7 +74,38 @@ class Chat extends Widget<Properties> {
   _renderMessageBox(): void {
     const $messageBox = $('<div>').appendTo(this.element());
 
-    this._messageBox = this._createComponent($messageBox, MessageBox, {});
+    const configuration: MessageBoxProperties = {
+      onMessageSend: (text) => {
+        this._messageSendHandler(text);
+      },
+    };
+
+    this._messageBox = this._createComponent($messageBox, MessageBox, configuration);
+  }
+
+  _createMessageSendAction(): void {
+    this._messageSendAction = this._createActionByOption(
+      'onMessageSend',
+      { excludeValidators: ['disabled', 'readOnly'] },
+    );
+  }
+
+  // eslint-disable-next-line max-len
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+  _messageSendHandler(action: any): void {
+    const { text, event } = action;
+    // @ts-expect-error
+    const { user } = this.option();
+
+    const message: Message = {
+      timestamp: String(Date.now()),
+      author: user,
+      text,
+    };
+
+    this.renderMessage(message, user);
+    // @ts-expect-error
```

Comment:
This error occurs because, according to your types, you expect that the properties of the message object will be passed into the event's data directly. Let's fix this, and also consider whether we should use Partial in typing on line 23.

Reply:
We're going to fix that later. We have a separate task for that